glide.yaml

```diff
@@ -1,7 +1,7 @@
 package: github.com/uber/tchannel-go
 import:
 - package: github.com/apache/thrift
-  version: master
+  version: ~0.9
```

Comment:
Note I don't think apache is going to be doing any updating, like I can set this to 0.9.3 if we don't support 0.9.1 or 0.9.2, but I don't think there will be a 0.9.4.